models/_sector/rebase_events/ethereum/rebase_ethereum_events.sql

```diff
@@ -0,0 +1,40 @@
+{{ config(
+
+        schema = 'rebase_ethereum',
+        alias ='events',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],
+        unique_key = ['tx_hash', 'evt_index'],
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "sector",
+                                    "rebase",
+                                    \'["hildobby"]\') }}'
```

Comment:
We have a new format for the `post_hook`:
```suggestion
        post_hook='{{ expose_spells(blockchains = \'["ethereum"]\',
                                    spell_type = "sector",
                                    spell_name = "rebase",
                                    contributors = \'["hildobby"]\') }}'
```